src/components/News/News.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import NewsCard from './NewsCard'
+import styles from './News.module.scss'
+
+const News = ({ data }) => {
+  return (
+    <div className={styles.items}>
+      {data.map(item => (
+        <NewsCard
+          key={`${item.title} ${item.description}`}
```

Comment:
Why so hard logic for a key. If items will not be changed, you can use index without any performance issues. @DmitriVanGuard Your mention?